contrib/azure/arm-template.json

```diff
@@ -87,6 +87,13 @@
       "metadata": {
         "description": "Version of CoreOS to deploy."
       }
+    },
+    "storageAccountType": {
+      "type": "string",
+      "metadata": {
+        "description": "Storage account type. Can be Premium_LRS (recommended) or Standard_LRS"
+      },
+      "defaultValue": "Premium_LRS"
```

Comment:
Good point on that - I thought of allowedValues after I pushed this, and wasn't sure it was worth doing. But I think it is.